src/table/resizer/index.tsx

```diff
@@ -35,6 +35,7 @@ export function Resizer({
   showFocusRing,
   onFocus,
   onBlur,
+  resizeDirection = 'right',
```

Comment:
Is the change to resizer related to the sticky-columns feature? Can it be shipped separately for the ease of review and testing?

Reply:
File was committed in mistake - after testing the resize handle on the left side of the last sticky column, we decided not to add this feature. 

- [ ] TODO: Remove this file